### PR TITLE
Feature/frontend

### PR DESCRIPTION
**Pull request by mtliendo**
Basic bootstrap-starter for the site using React.  Includes React-Router for front end routing. Webpack and babel for code bundling and transpiling, respectively.  Server.js and Webpack.config.js are extensively commented to help those new understand data flow.  Not meant to be an end-all starter (see comments in index.html).  Simply used to help visualize a layout. 

All necessary files needed to run are in the dist folder.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//setup basic server using express
+
+//require in modules from package.json
+const express = require('express')
+const app = express()
+
```
Comment (neorix11) [inline server.js]: Why not have `const PORT = process.env.PORT || 3000`?

**Comment by mtliendo** [comment]
The site is hosted on GitHub pages, which (unless I'm mistaken) only allows serving static content.

**Comment by ghost** [comment]
Will look at this tonight/tommorow morning. Am commenting as a reminder to myself.

**Comment by ghost** [comment]
A good start imo.  At the very minimum I think you should include documentation on how to build the project and remove the "built" files from the repo.  I understand how to get it going, but the javascript newbies likely will not, and they seem to be our target when it comes to docs.

**Pull request closed without merging**